src/lib/capabilities.tsx

```diff
@@ -142,7 +144,7 @@ export const getCapabilities = (
     formatArticle: !isViewer(),
     /* editor */
     editArticle: !isViewer(),
-    editMetadata: !(isViewer() || isAnnotator()),
+    editMetadata: !(isViewer() || isProofer()),
```

Comment:
why remove Annotater, as he can't change any model. so now the backend will allow it!